libraries/Microsoft.Bot.Builder.Dialogs/Prompts/IPromptCultureModel.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace Microsoft.Bot.Builder.Dialogs.Prompts
+{
+    /// <summary>
+    /// Culture model used in Choice and Confirm Prompts.
+    /// </summary>
+    public class IPromptCultureModel
```

Comment:
This would be better as a standard class ("PromptCultureModel"). When it's an interface like this, we can't add to it in the future w/o breaking compat.

Reply:
@cleemullins Understood. Chalk it up to me still being C#-dumb. Is this the direction we want to go with this, then: de-coupling from Text-Recognizers a little and using our own Culture Models? If so, I'll make this change tomorrow and do a PR for the JS side.

Reply:
@cleemullins Went ahead and made the change. I will interpret a Merge of this PR as approval of the changes and a request to create a parity PR for the JS side.